### ISSUE TITLE
AV when selecting object while dragging another one

### ISSUE THREAD

**Issue by ghost** [havlicekp/form-designer#3]
![image](https://user-images.githubusercontent.com/22324695/58722769-f94d0c80-83e0-11e9-8c3c-5f3f0fa68438.png)

**Comment by havlicekp** [comment]
Hello Andrew, can you explain more in detail what happened? How did you select an object while dragging one? Since while dragging a control your left mouse button is down and selecting another object shouldn't be possible? Was it by using TAB key?

**Comment by ghost** [comment]
> Hello Andrew, can you explain more in detail what happened? How did you select an object while dragging one? Since while dragging a control your left mouse button is down and selecting another object shouldn't be possible? Was it by using TAB key?

No, i've just released and pressed LMB by using RMB + LMB combination, i had a similar bug, to solve the problem i used ignoring parent messages, when the actual selected control is the same as proposed to select control.
